src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,273 @@
+import { cva, VariantProps } from "class-variance-authority";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const shellStylesOuter = cva(
+  "fixed inset-0 mx-auto mx-auto flex w-full max-w-4xl justify-center rounded-lg px-4 px-4 sm:px-6 sm:px-6 md:px-8 md:px-8",
+  {
+    variants: {
+      intent: {
+        primary:
+          "border-solid border-2 border-zinc-200 bg-white dark:bg-zinc-900",
+        secondary:
+          "border-solid border-2 border-brand-100 bg-brand-300 dark:bg-zinc-900",
+      },
+    },
+    defaultVariants: {
+      intent: "primary",
+    },
+  }
+);
+
+const shellStylesInner = cva(
+  "mx-auto mt-4 mb-16 w-full max-w-4xl px-4 sm:px-6 md:mt-16 md:px-8",
+  {
+    variants: {
+      intent: {
+        primary: "border-x-2 border-zinc-200 bg-white dark:bg-zinc-900",
+        secondary: "border-x-2 border-brand-100 bg-brand-300 dark:bg-zinc-900",
+      },
+    },
+    defaultVariants: {
+      intent: "primary",
+    },
+  }
+);
+
+const textStyles = cva("", {
+  variants: {
+    intent: {
+      primary: "text-zinc-500 dark:text-zinc-100",
+      secondary: "text-zinc-100 dark:text-zinc-100",
+    },
+  },
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const textCardStyles = cva("", {
+  variants: {
+    intent: {
+      primary: "text-zinc-400 dark:text-zinc-400",
+      secondary: "text-zinc-100 dark:text-zinc-100",
+    },
+  },
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const hrefStyles = cva("", {
+  variants: {
+    intent: {
+      primary: "text-blue-300 dark:text-blue-300",
```

Comment:
In general, don't use any colors other than `zinc` or `brand`. Not a hard and fast rule, but this way we can customize the colors easily in child projects